contracts/swappa/ISwappaPairV1.sol

```diff
@@ -8,4 +8,12 @@ interface ISwappaPairV1 {
 		address to,
 		bytes calldata data
 	) external;
+
+	// Get the output amount in output token for a given amountIn of the input token, with the encoded extra data.
+	// Output amount is undefined if input token is invalid for the swap pair.
+	function getOutputAmount(
+		address input,
```

Comment:
can you add 'address output' here too? 
output address will be needed for multi-token pools

Reply:
yeah need it for symmetric